tests/utils/tools.php

```diff
@@ -630,31 +633,31 @@ function configureFailPoint(MongoDB\Driver\Manager $manager, $failPoint, $mode,
         $doc["data"] = $data;
     }
 
-    $cmd = new MongoDB\Driver\Command($doc);
-    $result = $manager->executeCommand("admin", $cmd);
-    $arr = current($result->toArray());
-    if (empty($arr->ok)) {
-        var_dump($result);
-        throw new RuntimeException("Failpoint failed");
+    try {
+        $cmd = new MongoDB\Driver\Command($doc);
+        $result = $manager->executeCommand("admin", $cmd);
+    } catch (Exception $e) {
+        /* don't care if the fail point can't be set, it's MongoD version
```

Comment:
If you're now skipping tests when you know a failpoint isn't available, wouldn't you still want to keep the original behavior here (`var_dump()` and throw on unexpected failure to set a failpoint)?

Reply:
Is it a programming error to call `configureFailPoint()` without first checking that the fail point is supported? I thought that's the reason we have the various skip functions and special behavior like reporting that failpoints are never supported on mongos.

Reply:
It's complicated. Because we need to set two failPoints for getMore, which one is available depends on the exact mongod version. We can either ignore the exception, or, do lots of more complex checking in ``failGetMore``. I've decided that this now "good enough" for what we need.